docs/examples/example_lime.ipynb

```diff
@@ -74,18 +74,19 @@
     "* showcase the features with highest global importance\n",
     "\n",
     "In line with this ambition, they define a notion of 'coverage' -- to\n",
-    "be maximized --as follows:\n",
+    "be maximized -- with respect to a set of explanations {math}`V`, as follows:\n",
     "\n",
     "```{math}\n",
-    "  c(V, W, \\mathcal{I}) = \\sum_{j=1}^{d} I[\\exists i \\in V: W_{i,j} > 0] \\mathcal{I}_j\n",
+    "  c(V, W, \\mathcal{I}) = \\sum_{j=1}^{d} \\mathbb{I}\\{\\exists i \\in V: W_{i,j} > 0\\} \\mathcal{I}_j\n",
     "````\n",
     "\n",
     "where\n",
     "\n",
     "* {math}`d` is the number of features\n",
     "* {math}`V` is the candidate set of explanations to be shown to\n",
```

Comment:
On a high-level viewpoint, what is V? Is it more data or functions or response values or something entirely different?

Reply:
V is a subset of the explained dataset, this is the optimized variable which maximizes the coverage (information which a human can extract from a subset of explanations)

Reply:
I See. Maybe the sentence in the previous prompt (https://github.com/Quantco/metalearners/pull/21#discussion_r1651102718) is useful.

Reply:
Yes!